docs/gen-ai/aws-bedrock.md

```diff
@@ -0,0 +1,146 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: AWS Bedrock
+--->
+
+# Semantic conventions for AWS Bedrock operations
```

Comment:
please add a link to this (and other new AWS docs) from https://github.com/open-telemetry/semantic-conventions/blob/78c42c576a25743902192466cf7ff81889bf3630/docs/cloud-providers/aws-sdk.md#aws-service-specific-attributes